app/views/referee_mailer/reference_request_email.text.erb

```diff
@@ -0,0 +1,24 @@
+Dear ((name of referee)),
+
+# Please provide a reference for <%= @candidate_name %>
+
+We’re contacting you from the Department for Education as <%= @candidate_name %> has asked you to provide a reference to support their teacher training application.
+
+We would be grateful if you could provide a reference as soon as possible.
+
+Please use this form to complete your reference:
+<%= @google_form_url %>
+
+We’ll send you a reminder in a few days time.
+
+If we don’t receive your reference within 10 working days from the date of this email, we’ll ask <%= @candidate_name %> to suggest another referee.
+
+# Privacy notice
+
+<%= @candidate_name %> put us in touch with you so that you can provide a reference for them.
+
+The Department for Education and the teacher training provider(s) the candidate has applied to have a ‘legitimate interest’ under data protection legislation to hold your data for the period needed to process the candidate’s application.
+
+We’ll only use your data for this purpose and will delete it when no longer needed for this purpose.
+
+We currently use secure G Suite tools to process reference data. Your data will only be accessed by the staff who need it to process reference data.
```

Comment:
Is "G suite" a bit jargony? Do we need that first sentence?

Reply:
Agree. This is based on a first draft by Emma F and I'll feedback.

Reply:
Section removed in ccc8ad282c5e3b3ae2f9c11e14953929d66e7381